modules/core/src/main/java/org/apache/ignite/internal/processors/cache/persistence/snapshot/IgniteSnapshotManager.java

```diff
@@ -722,6 +721,8 @@ public static String partDeltaFileName(int partId) {
         busyLock.block();
 
         try {
+            snpRmtMgr.stop();
```

Comment:
If we stop the remote manager first, shuld we also pass `onNodeLeft` to the rmpMgr before?

snpRmtMgr.onNodeLeft(leftNodeId);
restoreCacheGrpProc.onNodeLeft(leftNodeId);

WDYT?

Reply:
`restoreCacheGrpProc` must first mark the active process as failed (`opCtx0.err.compareAndSet`). `snpRmtMgr` handles this error.